src/py42/_internal/client_factories.py

```diff
@@ -65,10 +65,12 @@ def get_alerts_client(self):
             self._alerts_client = alerts.AlertClient(session, self._user_context)
         return self._alerts_client
 
-    def get_departing_employee_client(self):
+    def get_departing_employee_client(self, user_client):
         if not self._departing_employee_client:
             session = self._get_jwt_session(u"employeecasemanagement-API_URL")
```

Comment:
I would like to lazily load this session so that `detection_list_user_client`, `high_risk_employee_client`, and `departing_employee_client` all use the same underlying session.

Reply:
Done changes.